### PR TITLE
Update octave to 3.8.2-1

### PR DESCRIPTION
**Pull request by jawshooah**
The Octave installer also installs a bunch of macports packages.
Not sure whether it's safe to delete these, but I included them for
review.

### CODE REVIEW

File: Casks/octave.rb [hunk 1]
```diff
@@ -1,18 +1,105 @@
...
   license :gpl
 
   pkg "Octave-#{version}.mpkg"
 
-  uninstall :pkgutil => [
-                         'org.macports.octave-next',
-                         'org.octave.cli-app',
-                         'org.octave.gui-app',
+  uninstall :quit    => 'org.octave.Octave',
+            :pkgutil => [
```
Comment (vitorgalvao) [inline Casks/octave.rb]: Double quotes are reserved for cases where string interpolation is needed, so these should be single quotes.

But why not simply `:pkgutil => 'org.macports.*'`?

Comment (jawshooah) [inline Casks/octave.rb]: I'm not sure that's safe, as future Casks may install other macports packages.

Comment (vitorgalvao) [inline Casks/octave.rb]: > Then again, future Casks may install some of these same macports packages, so it's not necessarily safe as-is either.

Agreed. Maybe just the octave-specific ones should remain.

Comment (jawshooah) [inline Casks/octave.rb]: Done in my last push.

**Pull request merged**